src/index.html

```diff
@@ -3,10 +3,42 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Calendar </title>
+    <link rel="stylesheet" href="./styles/bloks/main.scss">
```

Comment:
I guess we should put BEM blocks into bloks folder, not main.scss

Reply:
ok) i will correct it